### ISSUE TITLE
Scroll bar widths are too small

### ISSUE THREAD

**Issue by tjdennis** [ForkIssues/TrackerWin#607]
Can you change the scroll bar widths to use the Windows standard width?  When trying to grab the scrollbars on the right side of the Fork application, Windows keeps changing it to the resize cursor to resize the window instead of letting me scroll.  You have to slowly try and click on the left two pixels of the scroll bar in order to grab it instead of the window edge.

Thanks!

**Comment by Stanzilla** [comment]
Yes please! It also needs some color adjustments, in dark mode in diff-view it's almost impossible to see the bar when it gets small.
